values.yaml

```diff
@@ -138,6 +138,21 @@ mastodon:
     resources: {}
     # -- Affinity for all Sidekiq Deployments unless overwritten, overwrites .Values.affinity
     affinity: {}
+    # Rollout strategy to use when updating pods.
+    # Recreate is recommended for sidekiq pods for now because of possible data
+    # error between old/new versions. Please only use RollingUpdate if you are
+    # running a Mastodon version built after 2024-07-10.
```

Comment:
This is incorrect. `Recreate` will help reducing the number of retried jobs when updating if the code introduces a new job, but this has been the case for a long time.
`RollingUpdate` can be used for large clusters, if you are fine with some jobs being retried in the case above, but want to reduce the strain of starting all the pods at the same time. If using `RollingUpdate`, it is strongly suggested you also enable `readinessProbe`.